src/seedu/addressbook/ui/TextUi.java

```diff
@@ -75,7 +62,7 @@ private boolean isCommentLine(String rawInputLine) {
      * @return command (full line) entered by the user
      */
     public String getUserCommand() {
-        out.print(LINE_PREFIX + "Enter command: ");
+        out.print(Formatter.addLinePrefix() + "Enter command: ");
```

Comment:
You are still formatting the string here. Could perhaps move it to the formatter class.